1824 Silent Renew for OIDC is not working (only doc) (#1873)
SAP/luigi: plugins/auth/public/auth-oidc/README.md

})
```

If you want to use the silent token renewal feature, the `silent-callback.html` needs to be copied to a folder in your Luigi Core installation, 
which is the return path for the IdP provider, configured through the `redirect_uri` setting. The default location of `redirect_uri` is `/assets/auth-oidc/silent-callback.html`. 

> You must install oidc-client in your project as dev dependency:

Reviewer: ```suggestion
You must install `oidc-client` in your project as a dev dependency:
```

```javascript
npm i -save-dev copy-webpack-plugin
```

> Now you would need to import plugin files and oidc-client library in your project, using webpack:

Reviewer: ```suggestion
Then, you need to import the plugin files and `oidc-client` library in your project using webpack:
```
